packages/react/README.md

```diff
@@ -0,0 +1,48 @@
+# Clarity React Wrappers for Core Web Components
+
+Clarity Web Components is a suite of Web Components from the [Clarity Design System](https://clarity.design).
+
+## Installing Clarity Web Components
+
+1.  First, install the Clarity Core package from npm.
+
+    ```bash
+    npm install @clr/react --save
+    ```
+
+2.  Import desired Web Component into your JavaScript or TypeScript
+
+    ```typescript
+    import { CdsModal, CdsModalActions, CdsModalContent, CdsModalHeader } from '@clr/react/modal';
```

Comment:
Do we want people to have to know the details of our sub-components or would...

```
import * from '@clr/react/modal';
```

...work?

Reply:
I think those are all public components of modal, so the imports have to refer to each component since that is the same symbol that is in used in the JSX templates.

Reply:
What @coryrylan said is right. Unfortunately this explicit import of each component is needed.